### DISCUSSION TITLE
Is any MVVMCross support WinUI?

### DISCUSSION THREAD

**Discussion by sreekumarokp** [MvvmCross/MvvmCross#4338] [Q&A]
Is any MVVMCross support WinUI? If what is the version and other plugin required ?

**Comment by ivmirx** [comment]
It works with WinUI 2.x without problems, see the [UWP docs](https://www.mvvmcross.com/documentation/platform/uwp/universal-windows-platform-uwp?scroll=630) for details. But probably some adjustments are needed to get Windows App SDK and WinUI 3 working out of the box.

**Comment by DavidMarquezF** [comment]
I think it is in the works: https://github.com/MvvmCross/MvvmCross/pull/4316
